File: speak/src/main/java/org/sqlunet/speak/LexUnit.java
package org.sqlunet.speak;

import androidx.annotation.NonNull;

public class LexUnit extends Pronunciation
{
	final public String word;

	public LexUnit(final String word, final String ipa, final String variety)
	{
		super(ipa, variety);
		this.word = word;
	}

	@NonNull
	@Override
	public String toString()
	{
		return String.format("%s %s", word, super.toString());
	}
}

File: speak/src/main/java/org/sqlunet/speak/LexUnit.kt
/*
 * Copyright (c) 2023. Bernard Bou
 */
package org.sqlunet.speak

class LexUnit(val word: String, ipa: String, variety: String?) : Pronunciation(ipa, variety) {

    override fun toString(): String {
        return String.format("%s %s", word, super.toString())
    }
}
